.github/workflows/release.yml

```
@@ -0,0 +1,69 @@
+          convert assets/icon.png -resize 256x256 build/icons/256x256.png
+          cp assets/icon.png build/icons/512x512.png
+      
+      - name: Build .deb package
+        run: npm run dist
+      
+      - name: Get version from tag
+        id: get_version
+        run: echo "VERSION=${GITHUB_REF#refs/tags/v}" >> $GITHUB_OUTPUT
+      
+      - name: Create Release
+        uses: softprops/action-gh-release@v1
```

actionlint (1.7.9)

the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)